src/overlays/actors/ovl_Bg_Haka_Bombwall/z_bg_haka_bombwall.h

```diff
@@ -3,15 +3,18 @@
 
 #include "global.h"
 
+#define BGHAKABOMBWALL_GET_7F(thisx) ((thisx)->params & 0x7F)
+
 struct BgHakaBombwall;
 
 typedef void (*BgHakaBombwallActionFunc)(struct BgHakaBombwall*, PlayState*);
 
 typedef struct BgHakaBombwall {
-    /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x64];
+    /* 0x000 */ DynaPolyActor dyna;
+    /* 0x15C */ ColliderCylinder collider;
     /* 0x1A8 */ BgHakaBombwallActionFunc actionFunc;
-    /* 0x1AC */ char unk_1AC[0x4];
+    /* 0x1AC */ s8 csTimer;
+    /* 0x1AD */ UNK_TYPE1 unk_1AC[0x3];
 } BgHakaBombwall; // size = 0x1B0
```

Comment:
Struct padding probably
```suggestion
    /* 0x1AC */ s8 csTimer;
} BgHakaBombwall; // size = 0x1B0
```